hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/kafka/KafkaProperties.java

```diff
@@ -2,6 +2,8 @@
 
 public class KafkaProperties {
 
+    private String dc = "dc";
```

Comment:
Can we change the property name to `datacenter` ?
So we'd have `getDatacenter()` etc.

I'm afraid in the future people will ask: `Does dc stand for datacenter?` so it'll be clear. :)